core/enrollments/views.py

```
@@ -90,7 +90,24 @@ def post(self, request, contribution_id):
+            enrollment.payment_status = 'COMPLETED'
+            enrollment.payment_reference = 'FREE_ENROLLMENT'
+            enrollment.payment_method = 'FREE'
+            enrollment.save()
+            
+            return Response({
+                'status': True,
+                'message': 'Successfully enrolled in free contribution',
+                'data': {
+                    'enrollment_id': enrollment.id,
+                    'contribution_id': contribution.id,
+                    'contribution_title': contribution.title
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)